scripts/linters/pylint_extensions.py

```diff
@@ -1292,8 +1287,6 @@ def _compare_different_args(
         _compare_missing_args(
             params_with_doc, 'missing-param-doc',
             self.not_needed_param_in_docstring)
-        _compare_missing_args(
-            params_with_type, 'missing-type-doc', not_needed_type_in_docstring)
```

Comment:
Instead of removing these lines from here can we disable it using `.pylintrc`?

Reply:
Yup done -- thanks for the catch!